src/coprocessor/endpoint.rs

```diff
@@ -50,6 +50,7 @@ pub struct Endpoint<E: Engine> {
     batch_row_limit: usize,
     stream_batch_row_limit: usize,
     stream_channel_size: usize,
+    enable_batch_if_possible: bool,
```

Comment:
I prefer `enable_batch`

Reply:
no, since we haven't finished all batch executors and executions(it's a long work), so here we add an `if_possible`. That means when we will check if the current request can batch processed, if not possible, it will still be processed normally.